src/pages/ProductDetail/components/ProductInfo/ProductInfo.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import './ProductInfo.css';
+import profileimg from '../../../../assets/images/profile.png';
+import heartIcon from '../../../../assets/images/hearticon.png';
+
+function formatDate(createdDate) {
+  const date = new Date(createdDate);
+  const year = date.getFullYear();
+  const month = String(date.getMonth() + 1).padStart(2, '0');
+  const day = String(date.getDate()).padStart(2, '0');
+  
+  return `${year}.${month}.${day}`;
+}
+function ProductInfo( { item }) {
+  
+  return (
+    <>
+      <div className='info-container'>
+        <img className='product-img' src={item.images[0]} alt='제품 이미지'/>
+        <div className='product-text-container'>
+          <div className='product-text-box'>
+            <div className='main-box'>
+              <div className='name'>{item.name}</div>
+              <div className='price'>{item.price.toLocaleString()}</div>
+              <hr></hr>
+            </div>
+            <div className='sub-box'>
+              <label>상품 소개</label>
+              <div className='description'>
+                {item.description}
+              </div>
+              <label>상품 태그</label>
+              <div className='tag-list-container'>
+                {item.tags?.map((tagItem, index) => (
+                  <div className='tag'>
+                    <span key={index}>#{tagItem}</span>
+                  </div>
```

Comment:
P1:
key값은 해당 컴포넌트의 가장 부모에게 주어야합니다.

```suggestion
                  <div className='tag' key={index}>
                    <span>#{tagItem}</span>
                  </div>
```